logstash-core/lib/logstash/config/config_ast.rb

```diff
@@ -230,18 +232,18 @@ def compile_initializer
 
     def compile
       case plugin_type
-      when "input"
-        return "start_input(#{variable_name})"
-      when "filter"
-        return <<-CODE
+        when "input"
```

Comment:
That would be my editor (rubymine) sneaking in with its opinion. Good call, I'll re-indent this stuff.